driver-portal/src/Tests/ApiIntegrationTestBase.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+
+namespace Rsbc.Dmf.DriverPortal.Tests
+{
+    public abstract class ApiIntegrationTestBase
+    {
+        protected HttpClient _client { get; }
+        protected readonly IConfiguration Configuration;
+        protected const string CASE_API = "/api/Cases";
+
+        public ApiIntegrationTestBase(HttpClientFixture fixture)
+        {
+            _client = fixture.Client;
+            Configuration = fixture.Configuration;
+        }
+
+        protected async Task<T> HttpClientSendRequest<T>(HttpRequestMessage request)
```

Comment:
I would be tempted to refactor this later, to be `HttpClientSend(request).To<CaseDetail>()`. That will give us a little more flexibility and is slightly easier to read. I'm also tempted to see if we can share some integration test helper methods but for now, I'll wait until I understand the code more.